include/Util/Color.hpp

```diff
@@ -36,10 +47,10 @@ class Color : public glm::vec4 {
      */
     Color(const SDL_Color &color)
         : glm::vec4({
-              static_cast<float>(color.r) / 255.0F,
-              static_cast<float>(color.g) / 255.0F,
-              static_cast<float>(color.b) / 255.0F,
-              static_cast<float>(color.a) / 255.0F,
+              color.r,
+              color.g,
+              color.b,
+              color.a,
           }) {}
```

Comment:
i prefer to keep store data by 0.0~1.0

Reply:
As we discussed earlier, we agree to use 0-255 to store that value.

Reply:
sorry, seems like `SDL_Color` works with 0-255, i think your change is better.